book/src/building-from-source.md

```diff
@@ -48,6 +48,8 @@ RUSTFLAGS="-C target-feature=-crt-static"
 > the `runtime` directory within the user's helix config directory (more
 > [details below](#multiple-runtime-directories)).
 
+> 💡 The `runtime/sources` folder can be removed after building from source, which will save over 1GB of disk space.
```

Comment:
```suggestion
> 💡 The `runtime/grammars/sources` folder can be removed after building from source, which will save over 1GB of disk space.
```

Reply:
```suggestion
> 💡 The `runtime/sources` folder can be removed after building from source to free up disk space.
```

an exact(ish) figure might not age well, let's just say it'll save disk space

Reply:
I agree. It might be worth saying that it's "a significant amount of" disk space. I would ignore a message like that otherwise. In any case, having the note there will help in any form so thanks.